plugins/modules/pipx.py

```diff
@@ -163,6 +163,17 @@
   community.general.pipx:
     name: pycowsay
     state: absent
+
+- name: Install multiple packages from list
+  vars:
+    pipx_packages:
+      - pycowsay
+      - black
+      - tox
+  community.general.pipx:
+    name: "{{ item }}"
+    state: latest
+  with_items: "{{ pipx_packages }}"
```

Comment:
I think it's usually better to use `loop` than `with_items`, since the `ansible.builtin.items` lookup (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/items_lookup.html) sometimes does unexpected things. But in this case both work.